src/main/scala/org/ergoplatform/ErgoLikeContext.scala

```diff
@@ -115,14 +117,21 @@ class ErgoLikeContext(val currentHeight: Height,
 
 }
 
-object ErgoLikeContext {
+object ErgoLikeContext extends JsonCodecs {
   type Height = Int
 
-  val dummyPubkey: Array[Byte] = Array.fill(32)(0: Byte)
+  val dummyPubkey: Array[Byte] = GroupElementSerializer.toBytes(CryptoConstants.dlogGroup.generator )
```

Comment:
To fix NPE on in `CostingDataContext` on evaluation. Although it's weird that it happened only in `GetVar` test with invalid type. I should look into it again.

Reply:
Well, NPE is thrown during error message generation for GetVar invalid type. Since zero array decodes to infinity the `TestGroupElement.toString` throws NPE here https://github.com/ScorexFoundation/sigmastate-interpreter/blob/19a11e32efa884a850ae7d5a5b76e92daf53fed1/sigma-impl/src/main/scala/special/sigma/TestGroupElement.scala#L9-L9
I fixed it - https://github.com/ScorexFoundation/sigmastate-interpreter/pull/575/commits/f0a27776c4057119bf9e1419ae95835ffab1f38f
Do you want me to revert `dummyPubkey` back to infinity?

Reply:
There are two ways, 1) revert to minimise changes 2) analyse the code and prove that this change is NO HF.
I'm in favour of the second. But it is up to you.

Reply:
I will not be tempted by an easy way out. 😄  
Yay! My first NOHF comment - https://github.com/ScorexFoundation/sigmastate-interpreter/pull/575/commits/dcb3a5135016cbe2eecb66bb9148409f5cdf7ad9